File: Leetcode/Python/Finished/Easy/JewelsAndStones.py
class Solution:
    def numJewelsInStones(self, J, S):
        """
        returns the number of things that are in J in S.
        :param J: str
        :param S: str
        :return: int
        """
        count_j = {}
        for items in J:
            count_j[items] = 0
        for items in S:
            if items in count_j:
                count_j[items] += 1
        return sum(count_j.values())

File: Leetcode/C++/Finished/Easy/jewels_and_stones.cpp
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        int answer = 0;
        for (int i = 0; i < J.length(); i++) {
            for (int j = 0; j < S.length(); j++) {
                if (J.at(i) == S.at(j)) {
                    answer += 1;
                }
            }
        }
        return answer;
    }
};